podcasts/Onboarding/Login/LoginLandingView.swift

```diff
@@ -25,24 +24,14 @@ private struct LoginLandingContent: View {
 
     /// Reduce the header height to allow the buttons to fit for larger size categories
     private var useSmallHeader: Bool {
```

Comment:
We can delete this now

Reply:
💀 bdd6f5409346ff047c8ddf68de74757c64c7353c